lib/packwerk/configuration.rb

```diff
@@ -42,45 +42,10 @@ def initialize(configs = {}, config_path: nil)
       @root_path = File.expand_path(root)
       @package_paths = configs["package_paths"] || "**/"
       @custom_associations = configs["custom_associations"] || []
-      @load_paths = configs["load_paths"] || all_application_autoload_paths
+      @load_paths = configs["load_paths"]
       @inflections_file = File.expand_path(configs["inflections_file"] || "config/inflections.yml", @root_path)
 
       @config_path = config_path
     end
-
-    def all_application_autoload_paths
```

Comment:
@tomstuart the gist of it is that this method is getting moved out of `Configuration` and into a new file `ApplicationLoadPaths` because it has logic to load Rails files. This method is now testable in its new home.